platform/security/platform-security-core-api/src/main/java/ddf/security/permission/CollectionPermission.java

```diff
@@ -29,6 +29,16 @@
  */
 public class CollectionPermission implements Permission {
 
+    public static final String CREATE_ACTION = "create";
+
+    public static final String READ_ACTION = "read";
+
+    public static final String UPDATE_ACTION = "update";
+
+    public static final String DELETE_ACTION = "delete";
+
+    public static final String NO_ACTION = "no-action";
```

Comment:
If a permission check is being made then there is probably some kind of action it is just unknown.  No action seems misleading.